tests/test_packaging.py

```diff
@@ -6,14 +6,31 @@
 
 import pytest
 
+import opensafely
+
 
 BIN_DIR = "bin" if os.name != "nt" else "Scripts"
 
 project_fixture_path = Path(__file__).parent / "fixtures" / "projects"
 
 
+@pytest.fixture
+def version_file():
```

Comment:
`older_version_file` for a name perhaps?

Reply:
Oh yeah, that's better. I knew this name wasn't quite right!